include/rocksdb/advanced_options.h

```diff
@@ -704,6 +724,13 @@ struct AdvancedColumnFamilyOptions {
   // Dynamically changeable through SetOptions() API
   bool paranoid_file_checks = false;
```

Comment:
I think we should deprecate this option, talk about the equivalence in the new option, and how the two are reconciled until the removal of this one.